wp-content/themes/eighshift-task/src/blocks/assets/styles/parts/_core.scss

```diff
@@ -17,3 +17,19 @@ body {
   position: relative;
   overflow-x: hidden;
 }
+
+a {
```

Comment:
don't do this. this kind of general styling should be avoided. If you put any kind of lib in your project this will brake it. Style it per component.